docs/concepts/reftypes.mdx

```diff
@@ -0,0 +1,221 @@
+
+![OCI artifact with refererrs pointing to the artifact](/img/concepts/reftypes.png)
+
+## Creating a Reference Artifact
+
+OCI Artifacts may be stored in a content-addressable storage system or OCI Layout directory.
+For the purpose of this guide, we'll use the OCI Distribution Specification to
+create and list reference types.
+
+The creation of a reference artifact is a two-step process:
+
+1. **Step One**: Put an artifact, say Artifact A, using the PUT Manifest and
+blob upload APIs if the artifact has blob content.
+    - The prerequisite for a reference type is that the subject artifact must exist in the repository.
+
+2. **Step Two**: Put another artifact, say Artifact B, which refers to Artifact A.
+In this case, Artifact A is the 'subject' of Artifact B.
+    - This is done using the [push workflow][push-workflow] in the distribution specification.
+
+```mermaid
+sequenceDiagram
+    participant C as Client
+    participant R as Registry
+    C->>R: PUT /v2/<name>/manifests/<tag> Artifact A
+    R-->>C: 201 Created
+    C->>R: PUT /v2/<name>/manifests/<digest> Artifact B with reference to Artifact A
+    R-->>C: 201 Created
+```
+
+In this sequence, the client first uploads Artifact A to the registry.
+Next, the client uploads Artifact B to the same registry, but this time with a
+reference to Artifact A by specifying the digest of Artifact A as the subject.
+The association is now established in the registry.
+
+The following example shows the manifest of Artifact B with a reference to Artifact A in the `subject` field.
+
+```json
+ {
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType" : "application/example",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar",
+      "digest": "sha256:d2a84f4b8b650937ec8f73cd8be2c74add5a911ba64df27458ed8229da804a26",
+      "size": 12,
+      "annotations": {
+        "org.opencontainers.image.title": "attach/hello.txt"
+      }
+    }
+  ],
+  "subject": {
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "digest": "sha256:5e140a61e16155b30356685a6801e5250339bfb11370e70573d28d4ff2dc89cf",
+    "size": 477
+  },
+  "annotations": {
+    "org.opencontainers.image.created": "2023-08-05T06:58:38Z"
+  }
+}
+```
+
+## Listing Referrers
+
+Once an artifact has one or more referring artifacts, it's often necessary to
+list these referrers for various purposes such as tracking dependencies, updates
+or obtaining metadata of the artifact.
+The OCI [Distribution Specification][listing-referrers] defines an API endpoint
+(`/v2/<name>/referrers/<digest>`) for this purpose.
+
+If the registry does not support listing referrers then the [referrers tag schema][tag-schema]
+may be used to obtain and static list Index of refererres.
```

Comment:
```suggestion
may be used to obtain and statically list the Index of referrers. 
```

Reply:
update this to - 
If the registry does not support listing referrers, then the [referrers tag schema][tag-schema]
may be used to obtain a client created non-dynamic index of refererrers.